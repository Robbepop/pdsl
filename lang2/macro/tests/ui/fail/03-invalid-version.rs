#![feature(proc_macro_hygiene)]

use ink_lang2 as ink;

#[ink::contract(version = "foo")]
mod invalid_version {
    #[ink(storage)]
    struct InvalidVersion {}

    impl Noop {
        #[ink(constructor)]
        fn new(&mut self) {}

        #[ink(message)]
        fn noop(&self) {}
    }
}

fn main() {}
