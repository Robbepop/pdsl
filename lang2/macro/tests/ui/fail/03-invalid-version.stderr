error: couldn't match provided version as semantic version string: foo
 --> $DIR/03-invalid-version.rs:5:27
  |
5 | #[ink::contract(version = "foo")]
  |                           ^^^^^
